src/command_modules/azure-cli-vm/linter_exclusions.yml

```diff
@@ -0,0 +1,10 @@
+---
+# exclusions for the vm module
+
+vmss create:
+    parameters:
+        upgrade_policy_mode:
+            rule_exclusions:
+            - missing_parameter_help_rule
+
+...
```

Comment:
Like the new format!
What's the ... for at the end?

Reply:
indicates the end of a file.
It's optional, similar to the `---` at the start.